doc/client_lua_api.md

```diff
@@ -692,7 +692,7 @@ Call these functions only at load time!
 * `minetest.get_wielded_item()`
     * Returns the itemstack the local player is holding
 * `minetest.localplayer`
-    * returns LocalPlayer handle. See LocalPlayer below.
+    * Holds a reference to LocalPlayer object. See LocalPlayer below.
```

Comment:
Perhaps similar to `minetest.ui.minimap`?
`Reference to the LocalPlayer object. See `LocalPlayer` class reference for methods.`